translate/v23tov30/v23tov30.go

```diff
@@ -447,3 +447,75 @@ func translateDirectories(dirs []old.Directory, m map[string]string) (ret []type
 	}
 	return
 }
+
+// RemoveDuplicateFilesAndUnits is a helper function that removes duplicated files/units from
+// spec v2 config, since neither spec v3 nor the translator function allow for duplicate file
+// entries in the config.
+// This functionality is not included in the Translate function, but may be useful in cases
+// where configuration has to be sanitized before translation.
+// For duplicates, it takes ordering into consideration by taking the file/unit contents from
+// the slice with the highest index value, which is assumed to be the latest revision.
+// Multiple filesystems are not considered, it is assumed there is only one.
+// File appendices are not considered.
+// Unit dropins are concat'ed, i.e. appended if no duplicate dropin of the same name exists.
```

Comment:
This phrasing implies that the contents are concatenated into the same file, which isn't the case.

Reply:
clarified this a bit, please double check this is sufficient now